src/lightning_app/components/database/server.py

```diff
@@ -103,19 +95,20 @@ def run(self):
         self.db_filename = db_filename
         self.debug = debug
         self._models = models if isinstance(models, list) else [models]
-        self._client = None
+        self.token = token
```

Comment:
this should be `self._token`, so it's not exposed to the global state

Reply:
Done !